src/SymfonyRequirements.php

```diff
@@ -368,14 +378,14 @@ class_exists('PDO'),
      *
      * @return int - Converted size
      */
-    private function convertShorthandSize($size)
+    private function convertShorthandSize($size, string $infiniteValue = '-1')
```

Comment:
the requirements checker ismust currently be compatible with php 5.3. the string type breaks this.